Sources/SQLiteNIO/SQLiteDatabase.swift

```diff
@@ -0,0 +1,124 @@
+import NIOCore
+import NIOPosix
+import CSQLite
+import Logging
+
+public protocol SQLiteDatabase {
```

Comment:
Shall we put a Sendable on this now?

Reply:
I already have it locally; there's some changes I haven't pushed yet.

Reply:
In Salt-N-Pepa style - push it 😜